README.md

```diff
@@ -0,0 +1,107 @@
+# jsonp - vscode extension
+
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
+-   [Features](#features)
+-   [Extension Settings](#extension-settings)
+-   [Contribution](#contribution)
+-   [Feedback](#feedback)
+-   [About](#about)
+    -   [Maintainers](#maintainers)
+    -   [License](#license)
+
+## Getting Started
+
+This is a vscode extension supports enhanced JSON syntax (is called as `jsonp`)
+which is used by [python-jsonpreprocessor](https://github.com/test-fullautomation/python-jsonpreprocessor) package.
```

Comment:
Hi Ngoan,
syntax is "provided" by python-jsonpreprocessor.
Thank you,
 Thomas

Reply:
Hi Thomas,

I have updated it with new commit.

Thank you,
Ngoan